.github/workflows/build-images.yaml

```diff
@@ -30,14 +30,25 @@ jobs:
         id: add-branch-tag
         run: |
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Set Authorino version
-        id: authorino-version
+      - name: Set Authorino git sha
+        id: authorino-git-sha
         run: |
           if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          echo "version=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
```

Comment:
Do we need to remove the "v" prefix from the version? See e.g. https://github.com/Kuadrant/authorino-operator/pull/198/files#diff-6ec9d6ffc2c8ebd369edcc698ad16020762b6616d45849d4b539557e59563884R114